tls/extensions/s2n_client_supported_versions.c

```diff
@@ -71,6 +71,13 @@ int s2n_extensions_client_supported_versions_process(struct s2n_connection *conn
         uint8_t client_version_parts[S2N_TLS_PROTOCOL_VERSION_LEN];
         GUARD(s2n_stuffer_read_bytes(extension, client_version_parts, S2N_TLS_PROTOCOL_VERSION_LEN));
 
+        /* If the client version is outside of our supported versions, then ignore the value.
+         * S2N does not support SSLv2 except for upgrading connections. Since this extension is
```

Comment:
i don't think we have to worry about SSLv2 here - this list contains versions that client supports (which may include sslv2), what matters is finding the highest (valid) supported version client has, and the highest version we will negotiate, which we have done below.